src/Tizen.NUI/src/public/GLWindow.cs

```diff
@@ -111,17 +111,24 @@ public Size2D WindowSize
         {
             get
             {
-                var val = new Rectangle(Interop.GLWindow.GlWindow_GetPositionSize(swigCPtr), true);
+                global::System.IntPtr intPtr = Interop.GLWindow.GlWindow_GetPositionSize(swigCPtr);
+                var val = new Rectangle(intPtr, true);
                 Size2D ret = new Size2D(val.Width, val.Height);
+                val.Dispose();
 
                 return ret;
             }
             set
             {
-                var val = new Rectangle(Interop.GLWindow.GlWindow_GetPositionSize(swigCPtr), true);
-                Rectangle ret = new Rectangle(val.X, val.Y, value.Width, value.Height );
+                if (value == null)
```

Comment:
could you add exception comment ? 
<exception cref=xxx> xxx </exception>
(example) https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/Window.cs#L69

Reply:
Done
About warning CA1062, follow the comment of Mr. YeongJong Lee, remove it from current patch.